recipe/meta.yaml

```diff
@@ -23,28 +23,26 @@ requirements:
     - wheel
   run:
     - python
-    - numpy >=1.19.0
-    - backports.zoneinfo  # [py<39]
-    - matplotlib-base >=3.5.0
-    - pandas >=1.3.5
-    - scipy >=1.5.0
+    - numpy >=1.23.0
+    - scipy >=1.7.0
+    - pandas >=2.1.0
     - tzdata  # [win]
 
 test:
   imports:
     - mizani
-    - mizani.external
-    - mizani.tests
-    - mizani.colors
   requires:
     - pip
     - pytest
     - pytest-cov
   source_files:
-    - 'mizani/tests/*.py'
+    - "tests"
+    - pyproject.toml
   commands:
-    - pip check  # [not win]
-    - cd mizani && pytest tests -vv
+    - pip check
+    - pytest -vv # [not win]
+    # Skipping test as it is a bug in the code: Here is a PR fix https://github.com/has2k1/mizani/pull/46
```

Comment:
Should the source be patched with this? Since it affects windows and we are building this package with windows in mind?

Reply:
We could patch it but the problem is the code could probably use some more work to catch a few more cases.  What do you think @cbouss ?

Reply:
Fair enough. I am good with it as it is.